api/krusty/component_test.go

```diff
@@ -391,9 +392,6 @@ resources:
 nameSuffix: "-a"
 `),
 				writeC("comp-b", `
-resources:
```

Comment:
No longer permitted to call the same resources two times in each components due to the change to `resources` was transitioned to `ResMap` before applying components.